package.json

```diff
@@ -11,7 +11,7 @@
         "Arne Bahlo <arne@axiom.co>"
     ],
     "engines": {
-        "node": ">=16"
+        "node": ">=20"
```

Comment:
do we really want to do this for users? internally it's fine i think, but node 18 is still a supported lts

https://nodejs.org/en/about/previous-releases

Reply:
you are right, I will revert that.